spec/services/service_provider_seeder_spec.rb

```diff
@@ -79,6 +79,17 @@
     context 'when running in a production environment' do
       let(:rails_env) { 'production' }
 
+      context 'when %{env} is present in the config file' do
+        let(:deploy_env) { 'int' }
```

Comment:
Just pointing out that setting this here isn't strictly necessary, since it's set at the top of the file to the same value.

Reply:
good catch! I tweaked the test to set it to `dev` so it would be clear that it comes from the `deploy_env` setting.